vars/traefik.yml

```diff
@@ -1 +1,5 @@
-traefik_basic_auth: "{{ vault_traefik_basic_auth }}"
\ No newline at end of file
+traefik_basic_auth: "{{ vault_traefik_basic_auth }}"
+
+traefik:
```

Comment:
you can change this to just `traefik: {{ vault_traefik }}`

Reply:
I did just vault_traefik initially but kind of liked the more explicit way. I can change it to just vault_traefik though

Reply:
gonna amalgamate and delete all of vars/*.yml soon so want to keep the line count down

then we can document a copy of vars/secrets.yml in the readme  with none of the passwords showing